tests/nfv/run_performance_tests.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Run NFV Performance tests
+#
+# Maintainer: Jose Lausuch <jalausuch@suse.com>
+
+use base "opensusebasetest";
+use testapi;
+use strict;
+use utils;
+use lockapi;
+use mmapi;
+
+sub run {
+    select_console 'root-ssh';
+
+    record_info("Starting NFV Performance testing.");
+    assert_script_run('source /root/vsperfenv/bin/activate && cd /root/vswitchperf/');
+    assert_script_run('./vsperf --conf-file=/root/vswitchperf/conf/10_custom.conf --vswitch OvsVanilla phy2phy_tput',
+        timeout => 3600);
+    mutex_create("NFV_TESTING_DONE");
+}
+
+
+sub test_flags {
+    return {fatal => 1};
```

Comment:
does it have to be fatal?

Reply:
If this module doesn't work, the test can't be performed at all, so it makes sense to exit.